dmutils/logging.py

```diff
@@ -125,6 +125,10 @@ def format(self, record):
 
 
 class JSONFormatter(BaseJSONFormatter):
+    def __init__(self, *args, max_missing_key_attempts=5, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._max_missing_key_attempts = max_missing_key_attempts
```

Comment:
blimey this was just supposed to be an "emergencies only" knob to twiddle, didn't mean for it to spawn a constructor

Reply:
If we want to support this thing, we might as well do it properly? And pep/flake8 complains if you declare a variable not in the constructor, so 🤷‍♂️

Reply:
pfff `# qa`

Reply:
😱Ben will be horrified